dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04220RemoveDeleteInactiveClusterServersJob.java

```diff
@@ -22,9 +22,9 @@
     // Execution order of the queries is important
     private static final List<String> QUERIES = CollectionsUtils.list(
                     "DELETE FROM qrtz_excl_fired_triggers WHERE trigger_name = 'trigger23'",
-                    "DELETE FROM dotcms.qrtz_excl_cron_triggers WHERE trigger_name = 'trigger23'",
-                    "DELETE FROM dotcms.qrtz_excl_triggers WHERE trigger_name = 'trigger23' AND job_name = 'RemoveInactiveClusterServerJob'",
-                    "DELETE FROM dotcms.qrtz_excl_job_details WHERE job_name = 'RemoveInactiveClusterServerJob' AND job_class_name = 'com.dotmarketing.quartz.job.DeleteInactiveClusterServersJob'");
+                    "DELETE FROM qrtz_excl_cron_triggers WHERE trigger_name = 'trigger23'",
+                    "DELETE FROM qrtz_excl_triggers WHERE trigger_name = 'trigger23' AND job_name = 'RemoveInactiveClusterServerJob'",
+                    "DELETE FROM qrtz_excl_job_details WHERE job_name = 'RemoveInactiveClusterServerJob' AND job_class_name = 'com.dotmarketing.quartz.job.DeleteInactiveClusterServersJob'");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 190 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)